Dockerfile

```diff
@@ -24,6 +24,7 @@ ARG HOST_DIST=$TARGETOS-$TARGETARCH
 RUN adduser -D -u 12345 -g 12345 sm
 
 ADD --chown=sm:sm --chmod=0500 https://github.com/grafana/xk6-sm/releases/download/v0.0.3-pre/sm-k6-${TARGETOS}-${TARGETARCH} /usr/local/bin/sm-k6
+ADD --chown=sm:sm --chmod=0500 https://github.com/grafana/xk6-sm/releases/download/v0.4.0/sm-k6-${TARGETOS}-${TARGETARCH}-gsm /usr/local/bin/sm-k6-gsm
```

Comment:
Upgrading this past the above binary is intentional. These will be out of sync for a while until the default binary is updated and then we can drop the second one.

Reply:
Just to make sure this is intended, renovate will update *both* URLs:
https://github.com/grafana/synthetic-monitoring-agent/blob/ad3ffbbecf44565f74aefedc5dd3a91b6d138ad2/.github/renovate.json5#L32